index.bs

```diff
@@ -620,4 +620,5 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
   A special thanks to Mike West for his invaluable contributions to the initial
   version of this spec. Thanks to Brad Hill, Anne van Kesteren, Jonathan
   Kingston, Mark Nottingham, Sergey Shekyan, Dan Veditz, Eduardo Vela,
-  Tanvi Vyas, and Michal Zalewski for providing invaluable feedback.
+  Tanvi Vyas, Michal Zalewski and Robin Whittleton for providing invaluable
```

Comment:
This was sorted alphabetically. Please get in line :slightly_smiling_face:

Reply:
Amended that commit to line up 🙂